.gitignore

```diff
@@ -465,5 +465,6 @@ data/qdrant
 data/chatstore*
 
 # Java build
+java/.java-version
 java/**/target
```

Comment:
We cant modify files outside of the java directory, if you want changes outside of java, make them on the main branch